app.js

```diff
@@ -1,4 +1,5 @@
 const express = require('express');
+const endpoints = require('./endpoints.json');
```

Comment:
This method is vulnerable because require will only read the JSON once when the file is mounted and any subsequent requests will be sending a cached copy of the file. To ensure the most recent file is always read, we should be doing this asynchronously with fs readFile